engine/query-constants/src/test/java/io/deephaven/util/QueryConstantsTest.java

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QueryConstantsTest {
+
+    @Test
+    public void testMinFiniteFloat() {
+        assertEquals(Math.nextUp(-Float.MAX_VALUE), QueryConstants.MIN_FINITE_FLOAT, 0);
```

Comment:
I'd rather be explicit in comparison, with no delta to imply that there's math involved.
```suggestion
        final float calculated = Math.nextUp(-Float.MAX_VALUE);
        //noinspection SimplifiableAssertion
        assertTrue(calculated == QueryConstants.MIN_FINITE_FLOAT);
        assertEquals(
                Float.floatToRawIntBits(calculated),
                Float.floatToRawIntBits(QueryConstants.MIN_FINITE_FLOAT));
```

Reply:
Raw bits preserves NaN values, and A) we don't use those, plus B) these better not be NaNs anyway, so I tend to leave raw out of the picture. I like the more explicit checks though, will incorporate.

Reply:
We can drop `Raw` if you prefer, but since these aren't `NaNs` it seemed more explicit and safer. The values should be bitwise identical.